.gitignore

```diff
@@ -22,3 +22,4 @@ scripts/glext.h
 
 # vscode cmake plugin
 build/*
+.cache
```

Comment:
Why? What is this file? Please add a comment.

Reply:
I'll move this change to a separate commit and add a comment.